Assets/Shopify/UIToolkit/Shops/Generic/Classes/GenericMultiProductShop.cs

```diff
@@ -152,5 +161,9 @@ public class GenericMultiProductShop : MonoBehaviour, IMultiProductShop, IGeneri
             CloseButton.gameObject.SetActive(!canNavigateBack);
             BackButton.gameObject.SetActive(canNavigateBack);
         }
+
+        public void LoadMoreProducts() {
+            _controller.LoadMoreProducts (_productCache.Cursor);
```

Comment:
I'd like to rework this in another pr maybe... feels like it should just be what you said earlier as `LoadProducts(from: cursor, count: 5)` or whatever